kili/queries/asset/__init__.py

```diff
@@ -50,6 +53,7 @@ def assets(self, asset_id: Optional[str] = None, project_id: Optional[str] = Non
                                'skipped',
                                'status'],
                asset_id_in: Optional[List[str]] = None,
+               as_generator: bool = False,
```

Comment:
to put back in its place (we should not change the argument order unless there is a very good reason)

Reply:
(Else it is a breaking change for clients)